HOWTOCONTRIBUTE.rst

```diff
@@ -160,6 +160,39 @@ Preparing Pull Requests
 
    This command will run tests via the "pytest" tool against Python 3.6.
 
+#. Running the performance test suite
+
+Performance matters and it is worth considering whether your code has introduced
+performance regressions. *climpred* is starting to write a suite of benchmarking tests
+using `asv <https://asv.readthedocs.io/en/stable/>`__
+to enable easy monitoring of the performance of critical *climpred* operations.
+These benchmarks are all found in the ``climpred/asv_bench`` directory.
```

Comment:
They are actually just in the ``asv_bench`` directory. Should this get moved into the ``climpred`` directory? I'm not sure what convention is. If not, just replace this with ``asv_bench``

Reply:
the file structure is as in xarray